src/store/InMemoryBackingStore.cs

```diff
@@ -139,7 +139,7 @@ public void Subscribe(Action<string, object?, object?> callback, string subscrip
                 throw new ArgumentNullException(nameof(subscriptionId));
             if(callback == null)
                 throw new ArgumentNullException(nameof(callback));
-            subscriptions.Add(subscriptionId, callback);
+            subscriptions.TryAdd(subscriptionId, callback);
```

Comment:
I'm not sure I'm a big fan of silently failing. It's going to have big repercussions on the customer's code

Reply:
I see this now. Have updated the call to use `AddOrReplace` now so that the store is always updated.